src/java/com/test/TestQuiz.java

```diff
@@ -37,7 +37,15 @@ public void testCreateQuiz() throws Exception{
          assertEquals("Create fail.", true, a.createQuiz(quest, answer,id));
         
     }
-
+    
+    public void testUpdateQuiz() throws Exception {
+        assertEquals("Create fail.",true, a.UpdateQuiz(63,"hihi", "hoho"));
+    }
+    
```

Comment:
Create test for Delete Quiz in case: 
- qid is not exist in the database (done)
- qid exist. Check after run method delete quiz, qid mustn't be exist in the database, 
// Suggest: you can make query "select * from Quiz" to check it.